### PR TITLE
iles article

### PR DESCRIPTION
**Pull request by erondpowell**
This is the PR for the iles article! 👍

### CODE REVIEW

File: get-started-building-an-iles-website-with-directus/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+6. Your Directus Iles example is now running at <http://localhost:3000>.
+
+Now that you have the frontend and backend linked up, let's touch on what's happening in this project and where to find more sophisticated options.
+
+## Iles
+
+ [Iles](https://iles.pages.dev/guide/introduction) is a static-site generator (SSG) that provides great support for partial hydration. Use Vue and MDX to build your website, while enjoying the smooth development experience provided by Vite.js.
+
+Also, please note the Directus JavaScript SDK is pulled in under `examples > iles > src > login > directus.ts` _(find it [here](https://github.com/directus/examples/blob/main/iles/src/logic/directus.ts))_. If you'd like to learn more, checkout the [SDK documentation](https://docs.directus.io/reference/sdk/).
```
Comment (azrikahar) [inline get-started-building-an-iles-website-with-directus/index.md:60]:
```suggestion
Also, please note the Directus JavaScript SDK is pulled in under `examples > iles > src > logic > directus.ts` _(find it [here](https://github.com/directus/examples/blob/main/iles/src/logic/directus.ts))_. If you'd like to learn more, checkout the [SDK documentation](https://docs.directus.io/reference/sdk/).
```

File: get-started-building-an-iles-website-with-directus/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+![Directus Permissions](roles-and-permissions-20220222A.webp)
+
+For simplicity's sake, this Directus instance provided has [Public read permissions](https://docs.directus.io/getting-started/quickstart/#_6-set-role-public-permissions) activated for `articles` and `directus_users`. Find Permissions in your [local Directus Instance](http://localhost:8055/admin/settings/roles/public) under `Settings > Roles and Permissions`.
+
+Additionally, any number of Roles can be created and fully configured. Permissions are completely granular. Learn more about this in our documentation on [Users, Roles and Permissions](https://docs.directus.io/configuration/users-roles-permissions/).
+
+## APIs
+
+There is one simple request in this example worth noting, in `article.js`. If you'd like to see, here is a link to [article.js](https://github.com/directus/examples/blob/main/iles/src/logic/article.ts) on GitHub. However, please note that the Directus API is exhaustive for any database linked.
```
Comment (azrikahar) [inline get-started-building-an-iles-website-with-directus/index.md:76]:
```suggestion
There is one simple request in this example worth noting, in `article.ts`. If you'd like to see, here is a link to [article.ts](https://github.com/directus/examples/blob/main/iles/src/logic/article.ts) on GitHub. However, please note that the Directus API is exhaustive for any database linked.
```

**Review by azrikahar — COMMENTED** [review]
Understandably this is still in draft so you may already have these changes in mind, but let's attribute @ElMassimo with you instead of me in this article 😁

**Comment by ElMassimo** [comment]
Great! By the way, I'm ok with the current attribution of the article.

**Comment by ElMassimo** [comment]
@erondpowell Yeah, all the requests are using the `useArticles` helper, and providing different options as needed.

**Comment by erondpowell** [comment]
> @erondpowell Yeah, all the requests are using the `useArticles` helper, and providing different options as needed.

Sweet! @azrikahar @rijkvanzanten  if it looks good to you it looks good to me. Ready to fly.

**Review by rijkvanzanten — APPROVED** [review]

**Comment by ElMassimo** [comment]
Awesome! If you share your article on Twitter, you can tag `@ilesjs`, and I'll retweet it Thanks! 😃

**Pull request merged**